src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.cs

```diff
@@ -122,7 +122,7 @@ public NotifyFilters NotifyFilter
             set
             {
                 if (((int)value & ~c_notifyFiltersValidMask) != 0)
-                    throw new ArgumentException(SR.Format(SR.InvalidEnumArgument, "value", (int)value, typeof(NotifyFilters).Name));
+                    throw new ArgumentException(SR.Format(SR.InvalidEnumArgument, nameof(NotifyFilter), (int)value, nameof(NotifyFilters)));
```

Comment:
This "value" should be left as "value"... sorry if I missed this one before, too.  My point is that we shouldn't be changing the text that we're sending as part of this change... if nameof can be used to switch "something" to nameof(something), great, but otherwise we should leave the text as-is and not use nameof.